.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       # Run tests and build the binaries.
       - run:
           name: Run tests
-          command: go test -cover -v $(go list ./... | circleci tests split)
+          command: go test -cover -tags integration -v $(go list ./... | circleci tests split)
```

Comment:
would be good to run tests separately - integration and unit tests
i dont know what this change exactly does.

Reply:
To both comments, I'm concerned about perhaps losing the coverage data, don't know if it gets overridden. `-tags integration` includes unit tests either way.

Are we using publishing the coverage output somewhere ATM?

Reply:
Spoke with Namrata. Integration steps split to their own step.